src/pages/List/List.js

```diff
@@ -1,7 +1,38 @@
 import React from 'react';
+import styled from 'styled-components';
+import ListFilter from './ListFilter/ListFilter';
+import ListImages from './ListImages/ListImages';
 
 const List = () => {
-  return <div>List</div>;
+  return (
+    <Container>
+      <ContainerBox>
+        <ListFilter />
+        <ListWrap>
+          <ListImages />
+        </ListWrap>
+      </ContainerBox>
+    </Container>
+  );
 };
 
 export default List;
+
+const Container = styled.div`
+  background-color: ${props => props.theme.style.white};
+
+  display: flex;
+  justify-content: center;
+`;
+const ContainerBox = styled.div`
+  width: 60%;
+  margin-top: 50px;
+`;
+
+const ListWrap = styled.div`
+  width: 100%;
+  flex-wrap: wrap;
+  display: flex;
+  padding: 20px 0 20px 0;
+  justify-content: space-between;
```

Comment:
관련된 속성끼리 붙여 써주시는 게 좋습니다.
```suggestion
  display: flex;
  justify-content: space-between;
  flex-wrap: wrap;
  width: 100%;
  padding: 20px 0;
```